modules/runner/src/main/java/org/apache/ignite/internal/configuration/storage/LocalConfigurationStorage.java

```diff
@@ -46,17 +46,17 @@ public class LocalConfigurationStorage implements ConfigurationStorage {
     /** Prefix that we add to configuration keys to distinguish them in metastorage. */
     private static final String LOC_PREFIX = "loc-cfg.";
 
+    /** Key for the storage revision version. */
+    private static final ByteArray VERSION_KEY = new ByteArray(LOC_PREFIX + "$version");
```

Comment:
I wonder whether we should use version key outside of the config key range to make it absolutely impossible for a writer to cause a collision. If we still use the current key, we should probably validate that during write `VERSION_KEY` property is not written explicitly.

Reply:
> I wonder whether we should use version key outside of the config key range

This key starts with the common prefix on purpose, so that we can read the version in the `readAll` method in a single scan

> If we still use the current key, we should probably validate that during write VERSION_KEY property is not written explicitly

ok